cli/torcx-generator.go

```diff
@@ -117,3 +129,29 @@ func lowerProfiles(commonCfg *torcx.CommonConfig) ([]string, error) {
 	}
 	return lowerProfiles, nil
 }
+
+// vendorProfileFromDockerFlag returns the vendor profile name, if any,
+// implied by the contents of /etc/coreos/docker-1.12.
+func vendorProfileFromDockerFlag(commonCfg *torcx.CommonConfig) string {
+	flagPath := filepath.Join(filepath.Dir(filepath.Clean(commonCfg.ConfDir)), "coreos", "docker-1.12")
```

Comment:
This results in `/etc/torcx/coreos/docker-1.12` instead of what the comment says, it is intended?